examples/gno.land/r/demo/profile/profile_test.gno

```diff
@@ -0,0 +1,157 @@
+package profile
+
+import (
+	"std"
+	"testing"
+
+	"gno.land/p/demo/testutils"
+)
+
+const (
+	DisplayName        = "DisplayName"
+	Homepage           = "Homepage"
+	Bio                = "Bio"
+	Age                = "Age"
+	AvailableForHiring = "AvailableForHiring"
+	InvalidField       = "InvalidField"
+)
```

Comment:
You should not redeclare those constants here, as they are already defined in `profile.gno`.

I believe it should be seen as a "redeclaration" error. Could you please verify how Go manages this case and open an issue if we have a new bug?

Reply:
My apologies, I misread the text. This is the only implementation, so there is no bug.

It's good, but please move the constants to `profile.gno` and use them in `profile.gno` as well.

Reply:
- https://github.com/gnolang/gno/pull/1983/commits/fadbc59ffcdaa3b38bec6606df64f86712b5edca